tests/integration/actions/config/base.py

```diff
@@ -74,11 +77,22 @@ def test(self, request, tmux_session, step):
 
         if step.mask:
             # mask out some configuration that is subject to change each run
-            mask = "X" * 50
-            maskables = ["BECOME_PLUGIN_PATH", "CACHE_PLUGIN_CONNECTION", "COLLECTIONS_PATHS"]
-            for idx, line in enumerate(received_output):
-                if any(m in line for m in maskables):
-                    received_output[idx] = mask
+            maskables = [
+                "BECOME_PLUGIN_PATH",
+                "CACHE_PLUGIN_CONNECTION",
+                "COLLECTIONS_PATHS",
+                "DEFAULT_CALLBACK_PLUGIN_PATH",
+                "DEFAULT_LOCAL_TMP",
+            ]
+            # Determine if a menu is showing
+            mask_column_name = "CURRENT VALUE"
+            column_start = received_output[0].find(mask_column_name)
+            column_exists = column_start != -1
```

Comment:
help(str.find) returns the following:

```Help on method_descriptor:

find(...)
    S.find(sub[, start[, end]]) -> int
    
    Return the lowest index in S where substring sub is found,
    such that sub is contained within S[start:end].  Optional
    arguments start and end are interpreted as in slice notation.
    
    Return -1 on failure.
```

because it would take a doc look up I did added the named variable.

Reply:
I know that, it just looks ugly :)

Reply:
It looks magic (number) to me!

Reply:
Yeah. Even though I _know_ that it's likely coming from C with its limitations of the ancient times and missing exception-raising mechanism, I'd at least make it named like `SEARCH_FAILURE_CONSTANT = 1`. It's nice that WPS is able to catch some of such cases and maybe eventually we'll be more protected against similar code smells...

Reply:
One way of making this piece of code clearer would be
```python
try:
    column_start = received_output[0].index(mask_column_name)
except ValueError:
    pass  # no match, nothing to do
else:
    ...  # replacement logic here
```